File: src/main/kotlin/com/dai/dao/MusicMapper.java
package com.dai.dao;


import com.dai.bean.music.CloudMusic;
import com.dai.bean.music.MusicLibrary;
import org.apache.ibatis.annotations.Mapper;

import java.util.List;

@Mapper
public interface MusicMapper {


    void insertMusicLibrary(MusicLibrary musicLibrary);

    void insertCloudMusic(CloudMusic cloudMusic);

    List<MusicLibrary> getAllMusics();

    List<CloudMusic> getCloudMusics(int personId);

    List<String> getAllNames();

    Long getMusicId(String name);

    List<String> getNames(String personId);

    String getMusicPath(String name);
}

File: src/main/kotlin/com/dai/dao/MusicMapper.kt
package com.dai.dao

import com.dai.bean.music.MusicLibrary
import org.apache.ibatis.annotations.Mapper

/**
 * Created by dai on 2018/12/26.
 */
@Mapper
interface MusicMapper {

    fun getAllMusics(): List<MusicLibrary>
    fun insertMusicLibrary(musicLibrary: MusicLibrary)
}